components/Navbar.tsx

```diff
@@ -5,24 +5,24 @@ import { navLinks } from '../constants/navLinks';
 
 function NavBar() {
   return (
-    <div className="lg:mr-10 -mr-6">
-      <div className="block w-60">
-        <div className="text-3xl mb-4 font-semibold">Nathen Smith</div>
+    <div>
+      <div className="text-4xl mb-4 font-serif">Nathen Smith</div>
+      <div className="space-y-2 font-semibold text-2xl">
```

Comment:
To fix the responsiveness issue listed in the conversation, try

```suggestion
    <div className="max-w-3xl mx-auto">
      <div className="prose">
      <div className="text-4xl mb-4 font-serif">Nathen Smith</div>
      <div className="space-y-2 font-semibold text-2xl">
```